maven_install.json

```diff
@@ -651,6 +651,18 @@
                 "sha256": "66fdef91e9739348df7a096aa384a5685f4e875584cce89386a7a47251c4d8e9",
                 "url": "https://repo1.maven.org/maven2/org/hamcrest/hamcrest-core/1.3/hamcrest-core-1.3.jar"
             },
+            {
+                "coord": "org.jruby:jruby-complete:9.2.20.1",
```

Comment:
does "JRuby support version" become something we have to have an established policy on now as well? cc @devjgm

Reply:
> does "JRuby support version" become something we have to have an established policy on now as well? cc @devjgm

Most likely, yes.